share/locale/en.json

```diff
@@ -1,165 +1,165 @@
+    "share_via_onionshare": "OnionShare it",
     "gui_save_private_key_checkbox": "Use a persistent address\n(unchecking will delete any saved addresses)",
-    "gui_url_description": "<b>Anyone</b> with this link can <b>download</b> your files using the <b>Tor Browser</b>: <img src='{}' />",
-    "gui_url_label_persistent": "This share will not expire automatically unless a timer is set.<br><br>Every share will have the same address (to use one-time addresses, disable persistence in Settings)",
-    "gui_url_label_stay_open": "This share will not expire automatically unless a timer is set.",
-    "gui_url_label_onetime": "This share will expire after the first download",
-    "gui_url_label_onetime_and_persistent": "This share will expire after the first download<br><br>Every share will have the same address (to use one-time addresses, disable persistence in the Settings)",
-    "gui_status_indicator_stopped": "Ready to Share",
-    "gui_status_indicator_working": "Starting…",
-    "gui_status_indicator_started": "Sharing",
+    "gui_url_description": "<b>Anyone</b> with this OnionShare address your files using the <b>Tor Browser</b>: <img src='{}' />",
+    "gui_url_label_persistent": "This share will not expire automatically unless a timer is set.<br><br>Every subsequent share will be available at the same address (to use one-time addresses, turn off \"Persistence\" in the settings)",
+    "gui_url_label_stay_open": "Add a timer to automatically expire the share.",
+    "gui_url_label_onetime": "This OnionShare expires after one OnionSharing",
+    "gui_url_label_onetime_and_persistent": "This OnionShare expires after completion<br><br>Every subsequent OnionShare will be available at the same OnionShare address (to use one-time addresses, turn off \"Persistence\" in the settings)",
+    "gui_status_indicator_stopped": "Ready to OnionShare",
+    "gui_status_indicator_working": "Hyping…",
+    "gui_status_indicator_started": "OnionSharing",
     "gui_file_info": "{} Files, {}",
     "gui_file_info_single": "{} File, {}",
-    "info_in_progress_downloads_tooltip": "{} download(s) in progress",
-    "info_completed_downloads_tooltip": "{} download(s) completed",
-    "error_cannot_create_downloads_dir": "Error creating downloads folder: {}",
-    "error_downloads_dir_not_writable": "The downloads folder isn't writable: {}",
-    "receive_mode_downloads_dir": "Files people send you will appear in this folder: {}",
-    "receive_mode_warning": "Warning: Some files can hack your computer if you open them! Only open files from people you trust, or if you know what you're doing.",
+    "info_in_progress_downloads_tooltip": "{} underway",
+    "info_completed_downloads_tooltip": "{} completed",
+    "error_cannot_create_downloads_dir": "Error creating \"OnionShare\" folder: {}",
+    "error_downloads_dir_not_writable": "The \"OnionShare\" folder is write protected: {}",
```

Comment:
Actually this string and the previous should probably just refer to this as the "receive folder", rather than the "download folder" or the "OnionShare folder". So:

"Error creating receive folder: {}"
"The receive folder is write protected: {}"

Reply:
Is "inbox" out of question?